### PR TITLE
Cheatsheet for vcgencmd, Raspberry Pi's firmware command

### PR DESCRIPTION
**Pull request by iambibhas**
**What does your Instant Answer do?**
Shows a cheat sheet/reference of `vcgencmd`, Raspberry Pi's firmware command for querying the sensors etc

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Embedded Linux wiki](http://elinux.org/RPI_vcgencmd_usage).

**Why did you choose this data source?**
It's the preferred source from multiple places.

**Are there any other alternative (better) data sources?**
There is [this source](https://github.com/nezticle/RaspberryPi-BuildRoot/wiki/VideoCore-Tools) which I also consulted. It's not better than the other one.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Raspberry Pi enthusiasts.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No.

**Are you having any problems? Do you need our help with anything?**
No.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
selection_013

-----
IA Page: https://duck.co/ia/view/vcgencmd_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/vcgencmd.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    ],
+    "template_type": "terminal",
+    "section_order": [
+        "Utility",
+        "Sensors"
+    ],
+    "sections": {
+        "Utility": [
+            {
+                "key": "$ vcgencmd commands",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/vcgencmd.json]: Let's remove the leading `$`'s

**Comment by moollaza** [comment]
@iambibhas this looks pretty good! Let's just remove the `$` from the keys and I then this should be good for merge

**Comment by iambibhas** [comment]
@moollaza I didn't have them at first. Added them after seeing the image for different types of cheatsheet templates here - https://duck.co/duckduckhack/goodie_cheat_sheets

The image for type `terminal` has `$` before each command. -_-

**Comment by iambibhas** [comment]
@moollaza removed the `$`. Can be merged I think.

**Comment by moollaza** [comment]
>The image for type terminal has $ before each command. -_-

Sorry about that. I'll make a task to remove it there as well.

File: share/goodie/cheat_sheets/json/vcgencmd.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
+{
+    "id": "vcgencmd_cheat_sheet",
+    "name": "vcgencmd",
+    "description": "Raspberry Pi Firmware Command",
+    "metadata": {
+        "sourceName": "Embedded Linux Wiki",
+        "sourceUrl": "http://elinux.org/RPI_vcgencmd_usage"
+    },
+    "aliases": [
+        "vcgencmd"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/vcgencmd.json]: this isn't necessary. We use the filename as the default trigger, aliases are for any additional triggers.

**Comment by moollaza** [comment]
@iambibhas this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@iambibhas congrats! It's live: https://duckduckgo.com/?q=vcgencmd+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**